lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -143,12 +155,34 @@ for barclamp in $deactivation_list; do
 done
 
 
+# Stop chef-clients at the nodes, to ensure services shutdown in next step won't be started by chef
```

Comment:
Since I asked for another change in my previous comment, let me nitpick on the empty lines: there should be only one empty line here.